examples/quickstart/service.py

```diff
@@ -1,29 +1,20 @@
 from __future__ import annotations
 
-from transformers import pipeline
+from typing import List
```

Comment:
This is not necessary since we already uses future annotations. `list[str]` works